src/tuner/nccl_ofi_model.c

```diff
@@ -49,19 +49,18 @@ float nccl_ofi_tuner_compute_cost(struct nccl_ofi_tuner_model_params *params, st
 			num_internode_steps = 2 * dims->num_nodes;
 			latency = (num_internode_steps * net_lat)
 				  + (num_steps - num_internode_steps) * p2p_lat;
-			bw = params->internode_bw * params->num_rails * ofi_nccl_tuner_num_channels();
+			bw = params->internode_bw * params->num_rails;
 			break;
 
 		case NCCL_ALGO_NVLS_TREE:
 			latency = 2 * (p2p_lat + (log2(dims->num_nodes) * net_lat));
-			bw = NCCL_OFI_MIN(params->intranode_bw, (params->internode_bw * params->num_rails) / 2)
-			     * ofi_nccl_tuner_num_channels();
+			bw = NCCL_OFI_MIN(params->intranode_bw, (params->internode_bw * params->num_rails) / 2);
```

Comment:
This change (removing the `num_channels` parameter from bw for NVLS_TREE) causes a large deviation from observations for NVLSTree at large message sizes. We need to understand why.